frontend/src/routes/Governance/discovered/DiscoveredPolicies.tsx

```diff
@@ -0,0 +1,205 @@
+      },
+      {
+        header: t('Source'),
+        cell: (item: DiscoverdPolicyTableItem) => discoveredSourceCell(t, item.source),
+        sort: (a: DiscoverdPolicyTableItem, b: DiscoverdPolicyTableItem) =>
+          compareStrings(a.source?.type, b.source?.type),
+        search: (item: DiscoverdPolicyTableItem) => item.source?.type ?? '',
+        id: 'source',
+        exportContent: (item) => item?.source?.type ?? '-',
+      },
+      {
+        header: t('Violations'),
+        cell: (item: DiscoverdPolicyTableItem) => {
+          const { noncompliant, compliant, pending, unknown } = policyViolationSummary(item.policies)
+          const path = generatePath(NavigationPath.discoveredPolicyDetails, {
+            clusterName: item.policies[0].cluster ?? '',
+            apiGroup: item.policies[0].apigroup ?? '',
+            apiVersion: item.policies[0].apiversion ?? '',
+            kind: item.kind,
+            templateName: item.name,
+          })
+          if (noncompliant !== 0 || compliant !== 0 || pending != 0 || unknown !== 0) {
+            return (
+              <ClusterPolicyViolationIcons2
+                compliant={compliant}
+                compliantHref={path}
+                noncompliant={noncompliant}
+                violationHref={path}
+                pending={pending}
+                pendingHref={path}
+                unknown={unknown}
+              />
+            )
+          }
+          return '-'
+        },
+        sort: 'violations',
+        search: 'violations',
+        id: 'violations',
+        exportContent: (item: DiscoverdPolicyTableItem) => {
+          const { noncompliant, compliant, pending, unknown } = policyViolationSummary(item.policies)
+          return `${t('no violations: {{count}} cluster', { count: compliant })}, ${t('violations: {{count}} cluster', { count: noncompliant })}, ${t('pending: {{count}} cluster', { count: pending })}, ${t('unknown: {{count}} cluster', { count: unknown })}`
+        },
+      },
+    ],
+    [t]
+  )
+
+  const filters = useMemo<ITableFilter<DiscoverdPolicyTableItem>[]>(
+    () => [
+      {
+        id: 'violations',
+        label: 'Cluster violations',
+        options: [
+          {
+            label: t('No violations'),
+            value: 'no-violations',
+          },
+          {
+            label: t('Violations'),
+            value: 'violations',
+          },
+        ],
+        tableFilterFn: (selectedValues, item) => {
+          const { noncompliant, compliant } = policyViolationSummary(item.policies)
+
+          if (selectedValues.includes('no-violations') && noncompliant == 0) {
+            return true
+          } else if (selectedValues.includes('violations') && compliant > 0) return true
+          return false
+        },
+      },
+      {
+        id: 'kind',
+        label: t('Kind'),
+        options: [
+          { label: t('Configuration policy'), value: 'ConfigurationPolicy' },
```

Comment:
My previous code suggestion handles this, but in a previous discussion, we decided to use the kind value directly instead of sentence case as the filter label since this comes from the API.